modules/CRISPR.py

```diff
@@ -73,56 +36,50 @@ def main():
     logging.info(f'Number of descriptions: {len(description_df)}')
     logging.info(f'Number of cell/tissue annotation: {len(cell_lines_df)}')
 
-    ##
-    ## Process annotation
-    ##
-
     # Merging description with cell types and tissue:
-    tissue_desc = description_df.merge(cell_lines_df[['Name','Tissue']], left_on='tissue_or_cancer_type', how='inner', right_on='Tissue')
-    cell_desc = description_df.merge(cell_lines_df[['Name','Cancer Type']], left_on='tissue_or_cancer_type', how='inner', right_on='Cancer Type')
+    tissue_desc = description_df.merge(cell_lines_df[['Name', 'Tissue']], left_on='tissue_or_cancer_type', how='inner', right_on='Tissue')
+    cell_desc = description_df.merge(cell_lines_df[['Name', 'Cancer Type']], left_on='tissue_or_cancer_type', how='inner', right_on='Cancer Type')
 
     # Concatenating annotation:
     merged_annotation = pd.concat([tissue_desc, cell_desc], ignore_index=True)
 
     # Aggregating names accross disease/targets:
-    pooled_annotation = merged_annotation.groupby(['efo_id', 'tissue_or_cancer_type', 'method']).agg(
-            {'Name': lambda x: list(x)}).reset_index()
+    pooled_annotation = (
+        merged_annotation
+        .groupby(['efo_id', 'tissue_or_cancer_type', 'method'])
+        .agg(
+            {'Name': lambda x: list(x)}
+        )
+        .reset_index()
+    )
 
     # Updating columns:
     pooled_annotation = (
         pooled_annotation
-        .drop(['method',],axis=1)
+        .drop(['method', ], axis=1)
```

Comment:
For list literals of length 1 the trailing comma is not required (it is required for tuples, though)
```suggestion
        .drop(['method'], axis=1)
```